quickjs.c

```diff
@@ -52448,6 +52448,16 @@ static JSValue js_typed_array_get_byteOffset(JSContext *ctx, JSValue this_val)
     return js_uint32(ta->offset);
 }
 
+JSValue JS_NewTypedArray(JSContext *ctx, int argc, JSValueConst *argv,
```

Comment:
```suggestion
JSValue JS_NewTypedArray(JSContext *ctx, int argc, JSValue *argv,
```

Reply:
What would you recommend as a syntax to distinguish between pointers to `JSValue` that need to consume a reference and pointers to `JSValue` that don't ?

I still do not have a simple tool to perform a global check for reference count updates, but documenting this in the source code and header file would provide useful insights for other programmers.

Reply:
That is not a discussion for this PR really. NG doesn't have `JSValueConst`, so that shouldn't block this PR. Feel free to open an issue and let's discuss ideas!

Reply:
Yes of course, just a friendly reminder... not a blocker.